front/plugins/kea_api/script.py

```
@@ -0,0 +1,75 @@
+
+        mylog('verbose', [f'[{pluginName}] Querying Kea API at {url}'])
+
+        payload = {'command': 'lease4-get-all', 'service': ['dhcp4']}
+
+        response = requests.post(url, json=payload, auth=(user, password), timeout=max(1, timeout - 1))
+        response.raise_for_status()
+        data = response.json()
+
+        count = 0
+        for entry in data:
+            text = entry.get('text', '[API provided no text]');
```

Ruff (0.15.12)

[error] Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)